core/plannernotes.c

```diff
@@ -509,6 +557,13 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 			dp = dp->next;
 		}
 	}
+
+	/* For trimix OC dives, add ICD table here */
```

Comment:
Hmmm.. ok, now we finally could discuss if we want to have the ICD table really at the very end if the notes or if we want to have it after gas consumption but before warnings...
What's your opinion?